File: app/src/main/java/kh/edu/rupp/ite/mytravel/ui/activity/MainActivity.java
package kh.edu.rupp.ite.mytravel.ui.activity;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import kh.edu.rupp.ite.mytravel.R;
import kh.edu.rupp.ite.mytravel.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        setContentView(R.layout.activity_add_account);



        // button item action to fragment
        binding.bottomNavigationView.setOnItemReselectedListener(item -> {
            if (item.getItemId() ==R.id.button_home){

            }
        });
    }
}

File: app/src/main/java/kh/edu/rupp/ite/mytravel/ui/activity/MainActivity.kt
package kh.edu.rupp.ite.mytravel.ui.activity

import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import kh.edu.rupp.ite.mytravel.R

class MainActivity : AppCompatActivity() {

    @SuppressLint("MissingInflatedId")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Example: Start LoginActivity on clicking login button
        val loginButton = findViewById<View>(R.id.loginButton)
        loginButton.setOnClickListener {
            val intent = Intent(this, LoginActivity::class.java)
            startActivity(intent)
        }

        // Example: Start CreateAccountActivity on clicking create account button
        val createAccountButton = findViewById<View>(R.id.createAccountButton)
        createAccountButton.setOnClickListener {
            val intent = Intent(this, CreateAccountActivity::class.java)
            startActivity(intent)
        }
    }
}
